components/perf-tools/extrae/SPECS/extrae.spec

```diff
@@ -75,11 +73,13 @@ export LDFLAGS="$LDFLAGS -lz"
     --with-mpi-libs=$MPI_DIR/lib/release \
 %endif
 %if "%{compiler_family}" == "arm1"
-    CFLAGS="-O3 -fsimdmath -fPIC" CXXFLAGS="-O3 -fsimdmath -fPIC" FCFLAGS="-O3 -fsimdmath -fPIC" \
+    CFLAGS="${CFLAGS} -fsimdmath -fPIC -Wno-implicit-function-declaration" \
+    CXXFLAGS="${CXXFLAGS} -fsimdmath -fPIC" \
+    FCFLAGS="${FCFLAGS} -fsimdmath -fPIC" \
 %endif
     || { cat config.log && exit 1; }
 
-make %{?_smp_mflags}
+make %{?_smp_mflags} V=1
```

Comment:
What is `V=1` ? Looks like a debug leftover (verbose output)

Reply:
That is for verbose output, yes. It is for debug, but not really a leftover. I think it is easier to debug failed builds if the complete compiler command-line in the build-log is visible instead of just:
```
 CC path/to/file
```